docs/configuration/node-config.md

```diff
@@ -39,11 +39,8 @@ There are also other parameters that can be only defined by env variables:
 
 | Env variable | Description |
 | --- | --- |
-| `QW_S3_ENDPOINT` | Custom S3 endpoint. |
-| `QW_S3_MAX_CONCURRENCY` | Limit the number of concurent requests to S3 |
+| `QW_S3_MAX_CONCURRENCY` | Limit the number of concurrent requests to S3 |
```

Comment:
why not putting `QW_S3_MAX_CONCURRENCY` in s3 section like others?

Reply:
There was no corresponding config option, this is just an environment variable. but I have now moved it into that section as an env variable. I think we can create a config option for it if it makes sense like the others.